tests/generic_config_updater/templates/create_dhcp_forward_rule_both.json

```diff
@@ -0,0 +1,22 @@
+[
+    {
+        "op": "add",
```

Comment:
>add [](http://example.com/codeflow?start=15&length=3)

This actually replace the existing ACL_RULE table if already exsiting. Of course you can test, just want to make sure you intend to test it.

Reply:
It is applicable to all the testcase which "add" on a table.

Reply:
Yes, this is intended here and in other test cases.  I have one patch file (create_secondary_drop_rule.json) set up to perform an add to path "/ACL_RULE/DYNAMIC_ACL_TABLE|RULE_3" instead of just path "/ACL_RULE" - this rule is the only one that is ever added after the initial rule applications for a test.  Config rollback during teardown of test will restore whatever ACL_RULEs these patches replace initially